unicore/ask/service/schema.py

```diff
@@ -30,3 +30,36 @@ class QuestionSchema(colander.MappingSchema):
         validator=validators.question_type_validator)
     multiple = colander.SchemaNode(colander.Boolean(), default=False)
     options = Options(validator=validators.options_validator, default=[])
+
+
+class QuestionOptionSchemaPut(colander.MappingSchema):
+    uuid = colander.SchemaNode(
+        colander.String(),
+        validator=validators.option_uuid_validator, missing=None)
+    title = colander.SchemaNode(
+        colander.String(),
+        validator=validators.option_title_validator, missing=None)
+    short_name = colander.SchemaNode(
+        colander.String(),
+        validator=validators.option_short_name_validator,
+        missing=None)
+
+
+class OptionsPut(colander.SequenceSchema):
+    option = QuestionOptionSchemaPut()
+
+
+class QuestionSchemaPut(colander.MappingSchema):
+    title = colander.SchemaNode(
+        colander.String(),
+        validator=validators.question_title_validator, missing=None)
```

Comment:
I see that you only set attributes if they are not `None`, but for a `PUT` request all fields are required